paper-badge.html

```diff
@@ -181,7 +182,7 @@
 
       _updateTarget: function() {
         this._target = this.target;
-        this.async(this.notifyResize, 1);
+        requestAnimationFrame(this.notifyResize.bind(this));
```

Comment:
If you're asking why I'm binding this it is because rAF will give it a different context and `iron-resizable` does not care about bubbling but rather keeps track of the resizable elements firing non-bubbling `iron-resize` events.